src/lib/services/article.service.ts

```
@@ -7,9 +7,28 @@ import {
 
 const baseURL = 'https://cms.2077.xyz/api';
 
-export const fetchArticles = async (): Promise<ArticleMetadata[]> => {
+// Cache for storing fetched articles
+const cache = new Map();
+
+export const fetchArticles = async (category?: string, page: number = 1, limit: number = 10): Promise<ArticleMetadata[]> => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)